### ISSUE TITLE
Ampersand not converted to HMTL special `&amp;`

### ISSUE THREAD

**Issue by mvy** [novius-os/novius-os#14]
Using things stored into the `$wysiwyg['content']` I happen to notice that ampersand (&) you type in the back office page WYSIWYG are not converted to their HTML equivalent `&amp;` like other special characers (é as `&eacute;` etc...). 

This prevent the use of function like `loadHTML(htmlspecialchars_decode($wysiwyg['content']))` as it will fail on the ampersand. 

I am not sure this is under your control, but I wanted to signal it. I'll use "and" instead.

**Comment by felixgilles** [comment]
May be a a setting in Tinymce...
I watch monday

**Comment by felixgilles** [comment]
Hello Yves,

I just tested and I can not reproduce. I edited a page, I added "<&>" in the wysiwyg, I looked at the HTML source (via the button in toolbar) and I have "&lt; &amp; &gt;". I saved. In the database, it's still "&lt; &amp; &gt;" and it's the same in the front's source.

The amazing thing is that this functionality depends on the tinyMCE's option "encoding" :
http://www.tinymce.com/wiki.php/Configuration:encoding

However, this parameter is never initialized in Novius OS.
Is this related to the browser?

If you can reproduce it, can you give us the process and which browser / version / OS you use?

**Comment by mvy** [comment]
I checked the DB and the content seems ok. The output on the page too. Problem's coming from my use of subviews.

I just noticed that in my main view the output of `$wysiwyg['content']` variable gives the right answer i.e. : 

```
<h2>Titre</h2>
<p>Test des caract&egrave;res "sp&eacute;ciaux" &. 
```

However, after calling 

```
<?= \View::forge('ul_template::subviews/rightmenu', array('wysiwyg' => $wysiwyg['content']) ) ?>
```

the echo of `$wysiwyg` (in the subview) gives 

```
&lt;h2&gt;Titre&lt;/h2&gt;
&lt;p&gt;Test des caract&egrave;res &quot;sp&eacute;ciaux&quot; &amp;. &lt;/p&gt;
```

So I think the problem comes from my coding and the fact that `::forge` seems to transform its parameters. Since it seems I can get the original `$wysiwyg` variable in subview, what would be the right way to pass it through `::forge` without that nasty side effect?

Thanks for you attention.
